Sources/GRPC/Server.swift

```diff
@@ -447,3 +455,7 @@ private extension ServerBootstrapProtocol {
     }
   }
 }
+
+private func clamped<T>(_ value: T, lower: T, upper: T) -> T where T : Comparable {
+    return min(max(value, lower), upper)
+}
```

Comment:
This can be expressed a little more Swift-ly as:

```swift
extension Comparable {
  fileprivate func clamped(to range: ClosedRange<Self>) -> Self {
    return min(max(self, range.lowerBound), range.upperBound)
  }
}
```